pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -91,6 +91,8 @@ func (ed *eventDispatcher) dispatchEvent(pod *api.Pod, cgroupPath string, kind l
 		Pod: jsonPod,
 	}
 
+	// TODO: support more than one isolator return list of replies
```

Comment:
Didn't understand this comment (looks like you are already returning multiple replies)